llvm/test/DebugInfo/Symbolize/XCOFF/xcoff-symbolize-data.ll

```diff
@@ -0,0 +1,149 @@
+; RUN: llc -filetype=obj -o %t -mtriple=powerpc-aix-ibm-xcoff < %s
+; RUN: llvm-symbolizer --obj=%t 'DATA 0x60' 'DATA 0x61' 'DATA 0x64' 'DATA 0X68' \
+; RUN:   'DATA 0x90' 'DATA 0x94' 'DATA 0X98' | FileCheck %s
+
+; CHECK: bss_global
+; CHECK-NEXT: 96 4
+; CHECK-NEXT: /t.cpp:1
+; CHECK-EMPTY:
+
+; CHECK: bss_global
+; CHECK-NEXT: 96 4
+; CHECK-NEXT: /t.cpp:1
+; CHECK-EMPTY:
+
+; CHECK: data_global
+; CHECK-NEXT: 100 4
+; CHECK-NEXT: /t.cpp:2
+; CHECK-EMPTY:
+
+; CHECK: str
+; CHECK-NEXT: 104 4
+; CHECK-NEXT: /t.cpp:4
+; CHECK-EMPTY:
+
+; FIXME: fix the wrong size 152
```

Comment:
Are these wrong sizes corrected now? If so the FIXME comments should be removed.

Reply:
The wrong sizes are to be fixed in https://github.com/llvm/llvm-project/pull/67304 . My intention is to make this patch as the base and fix the wrong sizes in PR 67304 as the new test file is introduced in this PR.